proposal.rst

```diff
@@ -171,24 +188,27 @@ organisation(s)?**
 .. we don't have a problem with that, but it's helpful if we're aware of it
 .. so that we know how many backup choices we might need.
 
-FILLME
+No
 
-Your Project
-============
+Implement Bindings for Go
+==========================
 
 Motivations
 -----------
 
 **Why have you chosen this particular project?**
 
-FILLME
+My main areas of interest includes C++ and Go. They provide greater flexibility and concurrency.
```

Comment:
It isn't hugely important, but if you make a statement like "X provides greater Y", as you do here, this is meaningless without something to compare _to_. "X provides greater Y than Z" makes sense. It's almost never true that X provides greater Y than _everything else it could be compared to_, simply because of the enormous range of technologies out there. (Also, "concurrency" is a tricky beast. Do you mean concurrency within a single process, using threads and fibers; concurrency within a single machine or container which might also use multiple processes; concurrency within multiple machines in a cluster located closely within the network and/oir hardware topology; concurrency across arbitrary machines on the internet; and so on. I'm sure you know what you mean here, but it's always worth taking time to step back and think about whether you've provided enough detail to communicate that.)

A specific reason I'm pointing this out is that "<my favourite thing> provides greater <something or another>" is commonly seen online in people who get very close to one particular set of technologies, and believe they're better than everything else — but don't stay sufficiently up to date with other technologies to be able to know that for sure. So these kinds of statements online tend to be disregarded as being more about the person making them than about technology. It's good practice therefore to avoid statements like this, and instead be more precise about what you mean.